### YamlMime:UniversalReference
api_name: []
items:
- children:
  - azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode.bad_request
  - azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode.internal_server_error
  - azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode.resource_not_found
  class: azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode
  fullName: azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode
  inheritance:
  - inheritance:
    - type: builtins.object
    type: builtins.str
  - inheritance:
    - type: builtins.object
    type: enum.Enum
  langs:
  - python
  module: azure.cognitiveservices.personalizer.models.personalizer_client_enums
  name: ErrorCode
  summary: An enumeration.
  syntax: {}
  type: class
  uid: azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode
- class: azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode
  fullName: azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode.bad_request
  langs:
  - python
  module: azure.cognitiveservices.personalizer.models.personalizer_client_enums
  name: bad_request
  summary: Request could not be understood by the server.
  syntax:
    content: bad_request = 'BadRequest'
  type: attribute
  uid: azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode.bad_request
- class: azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode
  fullName: azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode.internal_server_error
  langs:
  - python
  module: azure.cognitiveservices.personalizer.models.personalizer_client_enums
  name: internal_server_error
  summary: A generic error has occurred on the server.
  syntax:
    content: internal_server_error = 'InternalServerError'
  type: attribute
  uid: azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode.internal_server_error
- class: azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode
  fullName: azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode.resource_not_found
  langs:
  - python
  module: azure.cognitiveservices.personalizer.models.personalizer_client_enums
  name: resource_not_found
  summary: Requested resource does not exist on the server.
  syntax:
    content: resource_not_found = 'ResourceNotFound'
  type: attribute
  uid: azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode.resource_not_found
references:
- fullName: azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode.bad_request
  isExternal: false
  name: bad_request
  parent: azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode
  uid: azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode.bad_request
- fullName: azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode.internal_server_error
  isExternal: false
  name: internal_server_error
  parent: azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode
  uid: azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode.internal_server_error
- fullName: azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode.resource_not_found
  isExternal: false
  name: resource_not_found
  parent: azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode
  uid: azure.cognitiveservices.personalizer.models.personalizer_client_enums.ErrorCode.resource_not_found
